tools/c7n_azure/c7n_azure/resources/event_hub.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('eventhub')
+class EventHub(ArmResourceManager):
+    """Event Hub Resource
+
+    :example:
+
+    Finds all Event Hub resources in the subscription.
+
+    .. code-block:: yaml
+
+        policies:
+            - name: find-all-eventhubs
+              resource: azure.eventhub
+
+    """
+
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Events']
+
+        service = 'azure.mgmt.eventhub'
+        client = 'EventHubManagementClient'
+        enum_spec = ('namespaces', 'list', None)
+        default_report_fields = (
+            'name',
+            'location',
+            'resourceGroup'
```

Comment:
We are trying to get better about default fields (We have an issue to go revisit them all) - you might consider adding `sku.name`